src/router/handlers/resourceHandler.ts

```diff
@@ -45,16 +45,27 @@ export default class ResourceHandler implements CrudHandlerInterface {
     }
 
     async create(resourceType: string, resource: any, tenantId?: string) {
-        await validateResource(this.validators, resourceType, resource, { tenantId, typeOperation: 'create' });
+        const strippedResource = validateXHTMLResource(resource);
+        await validateResource(this.validators, resourceType, strippedResource, { tenantId, typeOperation: 'create' });
```

Comment:
Can `validateXHTMLResource` be called in `validateReousrce` instead? It's a more logical organization of methods since it is a validation functionality, it also avoids calling it three times from different methods.

Reply:
I did originally have it there, the main issue I was running into was that it isn't possible to modify the resource in that method, which was the route I went because the package doesn't have a way I could find to throw errors instead. I will investigate further with security to see if there's a better choice out there.